README.md

```diff
@@ -1,29 +1,44 @@
 # HTML Build Tools
 
-This repository contains the tools and setup necessary for building the [HTML Standard](https://html.spec.whatwg.org/multipage/) from its [source](https://github.com/whatwg/html).
+This repository contains the tools and instructions necessary for building the [HTML Standard](https://html.spec.whatwg.org/multipage/) from its [source](https://github.com/whatwg/html).
 
-## Setup
+## Prerequisites
 
-1. Get a copy of [wattsi](https://github.com/whatwg/wattsi) and put the resulting binary in your PATH.
-1. Install `wget`.
-1. Check out [the source repo](https://github.com/whatwg/html) into this directory (by checking out into an empty folder then moving the contents here).
+Before building, make sure you have the following commands installed on your system.
 
-Then running build.sh inside this directory will generate the spec, multipage version, entities JSON file, and more.
+- `curl`
+- `egrep`
+- `git`
+- `grep`
+- `perl`
```

Comment:
> May also need to mention the perl libraries you need to have installed. My fresh Linux box did not have some XML one.

Yeah, so that’s not good. Given everything else we’re doing to make this thing as easy as possible, it’s gonna suck if contributors have to figure out how to install a Perl module that they don’t already have and don’t need for anything else.

But I think we don’t need to burden them with it. XML::Parser is need by just one of the perl scripts the build runs—the `.cldr-processor.pl` script, which takes as its input all the of the files in the `.cldr` subdir, and spits out just the one `cldr.inc` file.

The `cldr.inc` is just a **~25K file**, so rather than having N different contributors have to each build it themselves, we could either just check it into the repo here, and keep it versioned here, or put a copy up on whatwg.org that the build can each time get and cache locally.

As a bonus, that would also cut out both the `svn`, and the time the initial build and rebuilds need to (re)download 79MB of data from the Unicode CLDR subversion repo.

It’s true that the upstream Unicode CLDR sources change relatively often (on the order of one or more times a week), so it’d be some work to deal with keeping our `cldr.inc` file in sync with the upstream stuff. But we can of course automate a build of it that gets triggered whenever the upstream sources change, and that either puts the resulting `cldr.inc` on whatwg.org somewhere, or checks it into this repo.

Anyway, atm at least that’s what I think would be best. Let’s see if we can reach agreement on it.

Also for the record I did confirm that’s the only on non-"core" Perl module (that don’t actually ship with Perl itself) which our Perl scripts depend on. I hacked out a quick way to check:

```
for module in $(head .*.pl | grep "use "  | grep -v strict | grep ":" \
    | cut -c5- | cut -d" " -f1 | tr -d ";"); \
    do corelist $module; done
```

`corelist` is a command for doing this that ships with Perl I think.
I guess I should put that somewhere (other than here) for later. But anyway, here’s what it says:

```
Data for 2014-01-09
XML::Parser was not in CORE (or so I think)

Data for 2014-01-09
File::Path was first released with perl 5.001

Data for 2014-01-09
File::Basename was first released with perl 5

Data for 2014-01-09
Time::HiRes was first released with perl v5.7.3
```